worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,4 @@
 import math
-import random
-
-
-def random_point(width, height):
```

Comment:
I think it could be nice to have methods such as `random_point` or perhaps `random_land_point` but we could add it later if needed